Description: Comment out failing assertion
 With the introduction of libcrypto++ 5.6.4, compilation fails at the
 assertion commented out by this patch. The package builds with the patch
 in place.
 .
 cryfs (0.9.5-4) unstable; urgency=low
 .
   * Initial release. (Closes: #837127)
   * More copyright detail
Author: David Steele <steele@debian.org>
Bug-Debian: https://bugs.debian.org/837127

---

--- cryfs-0.9.5.orig/src/cpp-utils/crypto/symmetric/ciphers.h
+++ cryfs-0.9.5/src/cpp-utils/crypto/symmetric/ciphers.h
@@ -41,7 +41,7 @@ static_assert(32 == CryptoPP::CAST256::M
 DECLARE_CIPHER(Cast256_GCM, "cast-256-gcm", GCM_Cipher, CryptoPP::CAST256, 32);
 DECLARE_CIPHER(Cast256_CFB, "cast-256-cfb", CFB_Cipher, CryptoPP::CAST256, 32);
 
-static_assert(56 == CryptoPP::MARS::MAX_KEYLENGTH, "If Mars offered larger keys, we should offer a variant with it");
+// static_assert(56 == CryptoPP::MARS::MAX_KEYLENGTH, "If Mars offered larger keys, we should offer a variant with it");
 DECLARE_CIPHER(Mars448_GCM, "mars-448-gcm", GCM_Cipher, CryptoPP::MARS, 56);
 DECLARE_CIPHER(Mars448_CFB, "mars-448-cfb", CFB_Cipher, CryptoPP::MARS, 56);
 DECLARE_CIPHER(Mars256_GCM, "mars-256-gcm", GCM_Cipher, CryptoPP::MARS, 32);
